lib/contracts/contribution.js

```diff
@@ -20,24 +19,40 @@ class Contribution extends Base {
   }
 
   getById(id) {
-    id = ethers.utils.bigNumberify(id);
-
     return this.functions.getContribution(id)
-      .then((data) => {
+      .then(data => {
         return this.ipfs.catAndMerge(data, ContributionSerializer.deserialize);
       });
 
   }
 
-  getByContributor(contributor) {
-    return this.functions.balanceOf(contributor)
+  getByContributorId(contributorId) {
+    // return this.functions.balanceOf(address)
+    //   then(balance => {
+    //     count = balance.toNumber();
+    //
+    //     let contributions = [];
+    //
+    //     for (let index = 0; index <= count; index++) {
+    //       this.functions.tokenOfOwnerByIndex(address, index)
+    //         .then((id) => {
+    //           contributions.push(this.getById(id));
+    //         });
+    //     }
+    //
+    //     return RSVP.all(contributions);
+    //   });
+  }
+
+  getByContributorAddress(address) {
+    return this.functions.balanceOf(address)
       then((balance) => {
```

Comment:
It is missing a `.`

Reply:
I may be blind, but where exactly? Maybe you could turn it into a suggestion.